### PR TITLE
Create The-CodeINN.json

### PR DESCRIPTION
**Pull request by The-CodeINN**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/The-CodeINN.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "url": "https://twitter.com/thecodeinn",
+      "icon": "FaTwitter"
+    },
+    {
+      "group": "Tech Adventures",
+      "name": "Hashnode",
+      "url": "https://the-codeinn.hashnode.dev/",
+      "icon": "FaDev"
+    }
+  ],
```
Comment (sital002) [inline data/The-CodeINN.json:29]:
```suggestion
  ]
```
The trailing comma isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

File: data/The-CodeINN.json (hunk 1 above)
Comment (The-CodeINN) [reply data/The-CodeINN.json]: Resolved

**Review by SaraJaoude — APPROVED** [review]
Looks good to me - I will merge.

**Pull request merged**